sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -314,7 +338,9 @@ private[joins] object UnsafeHashedRelation {
       key: Seq[Expression],
       sizeEstimate: Int,
       taskMemoryManager: TaskMemoryManager,
-      isNullAware: Boolean = false): HashedRelation = {
+      isNullAware: Boolean = false,
+      isLookupAware: Boolean = false,
+      value: Option[Seq[Expression]] = None): HashedRelation = {
```

Comment:
Would `isLookedUp` be a better name for `value` ? Is there a way to force the Expression type to be a Boolean ?

Reply:
@agrawaldevesh - `value` here is the expression for value of key-value hash map. e.g. a HasedRelation(key: int, value: <int, double>). `value` here is the expression <int, double>. Similar to parameter `key` above.